Update TWONTO.ttl DOP65

mark-edney opened the pull request:
updated TWONTO with missing tags, verified with 13040 standard table.

mark-edney committed:
Update TWONTO.ttl

updated TWONTO with missing tags, verified with 13040 standard table.

TnyHng commented on OWL/TWONTO.ttl:
```diff
@@ -4979,6 +4991,8 @@ K. Power actuated PRV: a pressure relief valve actuated by an externally powered
                                      :is_equivalent_to_Avantis_category "Controller" ;
                                      :is_equivalent_to_Avantis_class "Controller" ;
                                      :is_equivalent_to_tag_code "CT" ;
+                                     :is_superclass_of_tag_code "Analysis Indicating Controller" ,
```
This appears to be a mistake (not a code)

mark-edney replied:
Corrected the tag reference, also add the tag PIC for process controller and FQ to flowmeter

TnyHng reviewed (commented):
Could you please confirm that you have you have done a "**reverse look up**", ensuring from the list of all classes and codes (exported from DB and 13040) all applicable ones have been incorporated?

mark-edney committed:
Update TWONTO.ttl

Corrected the tag reference, also add the tag PIC for process controller and FQ to flowmeter

TnyHng committed:
Merge branch 'main' into Missing_classes_and_tags_DOP65

TnyHng committed:
small change

TnyHng committed:
Merge branch 'main' into Missing_classes_and_tags_DOP65

TnyHng committed:
Merge branch 'Missing_classes_and_tags_DOP65' of https://github.com/TW-ASMP/TWONTO into Missing_classes_and_tags_DOP65

mark-edney commented:
> Could you please confirm that you have you have done a "**reverse look up**", ensuring from the list of all classes and codes (exported from DB and 13040) all applicable ones have been incorporated?

I can confirm I've completed the reverse lookup. Here are the up to date results from that: 
The following classes don't have matches:
| No. | Class        |
|-----|----------------------------|
| 1   | Chemical Feeder            |
| 2   | Control Loop               |
| 3   | Drive Mechanism            |
| 4   | Gas Collection             |
| 5   | Lighting                   |
| 6   | Meter                      |
| 7   | Peripheral                 |
| 8   | Plumbing                   |
| 9   | Pump                       |
|10   | Rigging Equipment          |
|11   | Software                   |
|12   | Storage                    |
|13   | Switch                     |
|14   | System Process Control     |
|15   | Transmitter                |

The following tags are verified from 13040 and don't have matches:
| Tag   | Entity Number | Verified by 13040                                |
|-------|----------------|--------------------------------------------------|
| AIT   | 2228           | Analysis Indicating Transmitter                 |
| BAT   | 51             | Battery (Bank) / Battery Charger                |
| ART   | 2              | Analysis Recording Transmitter                  |
| BRCH  | 1              | Breaching                                       |
| ASH   | 406            | Analysis Switch High, Smoke Detector            |
| F     | 761            | Filter                                          |
| FU    | 6              | Furnace                                         |
| IRG   | 9              | Irrigation System                               |
| FQIR  | 21             | Flow Totalizing Indicating Recorder             |
| MEQ   | 183            | Mechanical Equipment                            |
| PLQ   | 37             | Plumbing Equipment (not for Contracts -for WMS only) |
| PRTY  | 35             | Property                                        |
| SW    | 348            | Switch                                          |
| VENT  | 9              | Vent (not for Contracts -for WMS only)          |
